src/middleware/cache/index.ts

```diff
@@ -17,6 +22,7 @@ import type { MiddlewareHandler } from '../../types'
  * @param {string} [options.cacheControl] - A string of directives for the `Cache-Control` header.
  * @param {string | string[]} [options.vary] - Sets the `Vary` header in the response. If the original response header already contains a `Vary` header, the values are merged, removing any duplicates.
  * @param {Function} [options.keyGenerator] - Generates keys for every request in the `cacheName` store. This can be used to cache data based on request parameters or context parameters.
+ * @param {number[]} [options.cacheableStatusCodes=[200, 203, 204, 206, 300, 301, 404, 405, 410, 414, 501]] - An array of status codes that can be cached.
```

Comment:
Thank you. I think I need to change this too.

```suggestion
 * @param {number[]} [options.cacheableStatusCodes=[200]] - An array of status codes that can be cached.
```